config/bin_version_strings.cfg

```diff
@@ -300,6 +315,7 @@ ntp::"ntp[a-z][a-z]\ -\ .*\ NTP\ query\ program\ -\ Ver\.\ [0-9]\.[0-9]\.[0-9]+[
 ntp::"ntp[a-z]\ -\ .*\ NTP\ query\ program\ -\ Ver\.\ [0-9]\.[0-9]+\.[0-9]+$"
 ntp::"ntp[a-z]\ -\ .*\ NTP\ query\ program\ -\ Ver\.\ [0-9]\.[0-9]\.[0-9]+[a-z][0-9]+$"
 ntp::"ntp[pq]\ [0-9]\.[0-9]\.[0-9]+p[0-9]+$"
+openser::"^version:\ openser [0-9]\.[0-9]\.[0-9]+\ \(arm\/linux\)$"
```

Comment:
would this also work?
openser::"^version:\ openser\ [0-9]\.[0-9]\.[0-9]+\ \([a-z]\/[a-z]+\)$"

Reply:
I think there is a `+` missing.
I added this: 
`openser::"^version:\ openser\ [0-9]\.[0-9]\.[0-9]+\ \([a-z]+\/[a-z]+\)$"`